.github/workflows/schedule.py

```diff
@@ -1,49 +1,63 @@
+import argparse
 from datetime import (
     datetime,
+    timedelta,
 )
 import json
 import logging
 from operator import (
     itemgetter,
 )
 import subprocess
+import sys
 import zoneinfo
 
 tz = zoneinfo.ZoneInfo('America/Los_Angeles')
 
-now = datetime.now(tz)
-
 log = logging.getLogger('azul.github.schedule')
 
+one_week = 7  # in days
 
-def create_upgrade_issue():
-    # Pick any date for `start` and an issue will be created on that date, as
-    # well as every two weeks before and after, but only in the future. Note
-    # that this doesn't mean that the start date has to lie in the future.
-    start = datetime(2023, 11, 27, tzinfo=tz)  # Monday, …
-    if 0 == (now - start).days % 14 and now.hour == 9:  # … every other week, at 9am
-        template = '.github/ISSUE_TEMPLATE/upgrade.md'
-        front_matter, body = _load_issue_template(template)
-        labels, title = front_matter['labels'], front_matter['title']
-        process = subprocess.run([
-            'gh', 'issue', 'list',
-            f'--search=in:title "{title} {now.date()}"',
-            '--json=number',
-            '--limit=10',
-        ], check=True, stdout=subprocess.PIPE)
-        results = json.loads(process.stdout)
-        issues = set(map(itemgetter('number'), results))
-        if issues:
-            log.info('At least one matching issue already exists: %r', issues)
-        else:
-            subprocess.run([
-                'gh', 'issue', 'create',
-                f'--title={title} {now.date()}',
-                f'--label={labels}',
-                f'--body={body}'
-            ], check=True)
+
+def create_issue(gh_create_issue: list[str], gh_issue_lookup: list[str]) -> None:
+    process = subprocess.run(gh_issue_lookup, check=True, stdout=subprocess.PIPE)
+    results = json.loads(process.stdout)
+    issues = set(map(itemgetter('number'), results))
+    if issues:
+        log.info('At least one matching issue already exists: %r', issues)
     else:
-        log.info('Current time is outside of the configured schedule window')
+        subprocess.run(gh_create_issue, check=True)
+
+
+def gh_commands(template: str,
+                issue_date: datetime) -> tuple[list[str], list[str]]:
+    template = f'.github/ISSUE_TEMPLATE/{template}'
+    front_matter, body = _load_issue_template(template)
+    labels, title, assignees = [front_matter[x]
```

Comment:
```suggestion
    labels, title, assignees = [front_matter[k]
```

But I also think this is a bit much. If the tuple assignment is too long to fit on a line, tuple assignment is the wrong approach.